### PR TITLE
os-patch-boot

### PR DESCRIPTION
**Pull request by resmaa**
**3.1 Release Feature**

PCP-254: OS patch node reboot support

### CODE REVIEW

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1053#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
@resmaa: The following test **failed**, say `/retest` to rerun all failed tests:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
librarium_build | b6e62cf8e560dfb948de357a5840b39059471d97 | [link](http://34.120.121.77/view/gcs/capi-prow-artifacts/pr-logs/pull/spectrocloud_librarium/1053/librarium_build/1604790896795062272) | `/test librarium_build`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: content/docs/04-clusters/06-cluster-management/04-os-patching.md [hunk 1]
```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
...
 # Patch on boot
 During the cluster creation, while configuring the cluster (Cluster Configuration), the user can select “Patch OS on boot”. In this case, the operating system on all cluster nodes will be updated to the latest when the cluster VMs are initially deployed.
 
 |  **During Cluster Creation**       |
 | -----------------------------------|
-|Cluster Configuration -> "Patch on Boot" |
+|Cluster Configuration -> **Patch on Boot**|
+
+
+# Reboot If Required
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/04-os-patching.md:27]:
```suggestion
# Reboot if Required

```
This is one of the things that is annoying with title case headings, but the norm is to not title case words fewer than three words, such as the, as, in, to, if, and, and many more.

File: content/docs/04-clusters/06-cluster-management/04-os-patching.md [hunk 2]
```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
...
+
+Palette supports the **Reboot If Required** feature to control the system reboot as part of cluster upgrades. Some system upgrades will require a reboot to apply the changes to the cluster. You need to check the **Reboot if required** button to allow the reboot. If this option is unchecked, the system reboot will be restricted.
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/04-os-patching.md:29]:
```suggestion
Palette supports the **Reboot if Required** feature to control the system reboot as part of cluster upgrades. Some system upgrades will require a reboot to apply the changes to the cluster. You need to check the **Reboot if Required** checkbox to allow the reboot. If this option is unchecked, the system reboot will be restricted.
```
Fixed inconsistency

File: content/docs/04-clusters/06-cluster-management/04-os-patching.md [hunk 3]
```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
...
+
+|  **During Cluster Creation**       |
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/04-os-patching.md:31]: @resmaa I'm confused as to what we are trying to show with a table.

File: content/docs/04-clusters/06-cluster-management/04-os-patching.md [hunk 4]
```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
...
+| -----------------------------------|
+|Cluster Configuration ->**Reboot if required**|
 
 
 # Scheduled
-Besides patching on boot, the users also have the option to set a schedule for OS patching. Patching schedule can be set initially when creating a cluster as well as at any given point later. Following scheduling options are provided:
+Besides patching on boot, you have the option to set a schedule for OS patching. Patching schedule can be set initially when creating a cluster as well as at any given point later. Following scheduling options are provided:
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/04-os-patching.md:37]:
```suggestion
Besides patching on boot, you can set a schedule for OS patching. The patching schedule can be set initially when creating a cluster as well as at any given point later. The following scheduling options are provided:

```

File: content/docs/04-clusters/06-cluster-management/04-os-patching.md [hunk 5]
```diff
@@ -12,45 +12,51 @@ import InfoBox from 'shared/components/InfoBox';
...
 * Every two weeks at midnight
 * Every month on the 1st at midnight
 * Every two months on the 1st at midnight
-* Custom OS patch for an exact month, day, hour and minute of user’s choice
+* Custom OS patch for an exact month, day, hour and minute of your choice
 
 |  **During Cluster Creation**       |
 | -----------------------------------|
-|Cluster Configuration -> "OS Patching Schedule" |
+|Cluster Configuration -> **OS Patching Schedule** |
```
Comment (karl-cardenas-coding) [inline content/docs/04-clusters/06-cluster-management/04-os-patching.md:47]: @resmaa same feedback as above related to tables.

**Review by karl-cardenas-coding — CHANGES_REQUESTED** [review]
Feedback and suggestions provided

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1053#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by spectro-prow** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/spectrocloud/librarium/pull/1053#pullrequestreview-1226390303" title="Approved">karl-cardenas-coding</a>*, *<a href="https://github.com/spectrocloud/librarium/pull/1053#" title="Author self-approved">resmaa</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign 
You can assign the PR to them by writing `/assign ` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=spectrocloud%2Flibrarium).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/spectrocloud/librarium/blob/release-3-1/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**